reporter/internal/pdata/generate.go

```diff
@@ -43,6 +46,8 @@ func (p *Pdata) Generate(tree samples.TraceEventsTree,
 	// By specification, the first element should be empty.
 	stringSet.Add("")
 	funcSet.Add(funcInfo{nameIdx: stringSet.Add(""), fileNameIdx: stringSet.Add("")})
```

Comment:
Do we need a default entry in the function table at all? It's not documented in the proto (as for mapping, link and string tables).

Reply:
We should add the documentation in the proto I think. Native frames won't have a `funcInfo` and will fall back to the zero element in index 0.